### DISCUSSION TITLE
Disable events broadcasting

### DISCUSSION THREAD

**Discussion by mgdmiller** [JasperFx/marten#2687] [General]
Hello colleagues!

I broken my mind. Maybe I don't fully understand how it works. I have universal event, applied for multiple types, e.g. "ObjectCreated". When i append events to stream, invokes all Apply methods for all types. How to make the event apply only to a specific object and not to everyone who is "subscribed" to the event? And in general, is it possible?

**Comment by jeremydmiller** [comment]
Hey, can you ask this on Discord? Issues are reserved for bugs or enhancements please.

**Comment by mgdmiller** [comment]
Thank you for reply!
